uber/templates/preregistration/preregbase.html

```diff
@@ -112,23 +120,27 @@
             }
         };
         $(function () {
-            $.each([REG_TYPES, BADGE_TYPES], function (i, types) {
-                $.each(types.options, function (index, type) {
-                    $(types.row).append(
-                        $('<div class="col-sm-3"></div>').append(
-                            $('<a class="list-group-item"></a>').addClass(types.selector.substring(1)).click(function () {
-                                setBadge(types, index);
-                            }).append(
-                                $('<h4 class="list-group-item-heading"></h4>').html(type.title)
-                            ).append(
-                                $('<p class="list-group-item-text"></p>').html(type.description))));
```

Comment:
I wrapped this in an if statement because we were getting some (harmless but annoying) Javascript errors on pages which don't use this code, such as the checkout page.